### DISCUSSION TITLE
Opening a mouse homology pathway

### DISCUSSION THREAD

**Discussion by pep-marshall** [wikipathways/wikipathways-help#135] [Q&A] [answered]
Hello,
I would like to work with mouse homology pathways found here:
https://github.com/wikipathways/wikipathways-homology/tree/main/pathways/Mm

However I cannot open them in Pathvisio and receive an "error on line 24: attribute name "crossorigin" associated with element type.." message.

I have tried with several, eg. Human Rett Syndrome found here: <https://www.wikipathways.org/pathways/WP4312.html>
This opens fine if I download the .gpml and open in Pathvisio.

If I try the mouse homology pathway found here: <https://github.com/wikipathways/wikipathways-homology/tree/main/pathways/Mm/WP4312>
I get an error message.

Upon inspecting the files WP4312.gmpl is encoded in XML
WP4312_Mm.gpml is encoded in HTML and looks very different.

Can anyone help with opening the mouse pathway please?

**Comment by khanspers** [comment, marked as answer]
Hi @pep-marshall, I can't replicate the error you're seeing. Both the gpml downloaded from wikipathways.org (https://www.wikipathways.org/pathways/WP4312.html) and the gpml downloaded from the homology repo (https://github.com/wikipathways/wikipathways-homology/blob/main/pathways/Mm/WP4312/WP4312_Mm.gpml) open without a problem for me. 

For the gpml from the homology repo, make sure you click the "Download raw file" ![Screen Shot 2024-09-23 at 10 15 02 AM](https://github.com/user-attachments/assets/947909b1-4a1c-4008-8a04-1d6c8c7e417b) in the upper right of this page: 

https://github.com/wikipathways/wikipathways-homology/blob/main/pathways/Mm/WP4312/WP4312_Mm.gpml

**Comment by pep-marshall** [comment]
Thank you very much, clicking the link to view the code and then selecting "download as raw" downloaded a working .gpml file

I was right-clicking the link and "save as" which was saving as gpml. It's a peculiarity of GitHub I guess, I don't use it that much!

Thanks again
